### DISCUSSION TITLE
AVM Wireguard: nur die erste Route wird übernommen

### DISCUSSION THREAD

**Discussion by Tone866** [Freetz-NG/freetz-ng#1077] [Allgemein]
Wenn ich auf meiner 7490 mit aktuellem freetz-ng und fritzos ein VPN mit mehreren Routen aktivieren will, wird nur die erste Route übernommen.
Dieses Verhalten scheint ohne freetz-ng nicht aufzutreten.

In der FritzOS-GUI stehen folgende entfernten Netze im VPN eingetragen:
172.16.20.1/24, 172.64.0.0/13, 192.168.9.0/32

Wenn ich `ip r` ausführe, bekomme ich aber nur eine Route für das erste Netz (172.16.20.1/24) angezeigt.
Die anderen beiden Netze fehlen vollständig und werden auch nicht über den Tunnel geroutet.

**Comment by fda77** [comment]
Keine Ahnung, ich hab den minderwertigen AVM-Wireguard Kram nie genutzt. Funktioniert damit ipv6 mitlerweile komplett?
Du musst aber selbst suchen, es gibt um freetz nicht wirklich eine Community. Vielleicht ist ein anderes Forum aufschlussreicher
